config/labels/analyzers/clangsa.json

```diff
@@ -306,9 +317,6 @@
       "doc_url:https://clang.llvm.org/docs/analyzer/checkers.html#alpha-webkit-uncountedlocalvarschecker",
       "profile:extreme"
     ],
-    "altera-struct-pack-align": [
```

Comment:
Why did you remove this? Users who are using old clang versions will still have this checker.

Reply:
Not just old versions, I think this check is still part of the upstream distribution: https://github.com/llvm/llvm-project/blob/91f4650ebb057de7f6ed6d038985b2d53d144e15/clang-tools-extra/clang-tidy/altera/StructPackAlignCheck.cpp

(Then again, it is a very guidelines or platform specific check.)